### DISCUSSION TITLE
Cards Layout

### DISCUSSION THREAD

**Discussion by Alv-sys** [directus/directus#7182] [Feature Requests]
Playing with the Layout options of the collections, the "Cards" Layout, I realized when the title is longer than the space, it just turn into three points "...". Probably it should be nice to have by default the field always with the full title or if it is a problem the space, keeping some reasonable part of the string to recognize them better

![image](https://user-images.githubusercontent.com/47535920/128114044-1d3d6100-0e0e-49ea-a2cd-a234ec40ac5f.png)

**Comment by benhaynes** [comment]
Thanks!

@rijkvanzanten, is this because of the inline display templates? Wonder if/how we can avoid this.

**Comment by rijkvanzanten** [comment]
@Alv-sys could you open a new issue? Thanks!
